packages/documentation/src/stories/utilities/visibility/visibility.styles.scss

```
@@ -0,0 +1,19 @@
+@use '@swisspost/design-system-styles/core' as post;
+.visibility-box {
+  width: 100px;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: post.$yellow;
+  border: post.$white 2px solid  ;
+  border-radius: 0.5rem;
+}
+
+.visibility-example {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
```

Here you can use nested selectors so that the `.visibility-box` class is not shown in the code snippet, that way we only see the visibility classes changes.

```suggestion

.visibility-example {
  display: flex;
  align-items: center;
  gap: 0.5rem;
  
  > div {
    width: 100px;
    height: 100px;
    display: flex;
    align-items: center;
    justify-content: center;
    background-color: post.$yellow;
    border: post.$white 2px solid  ;
    border-radius: 0.5rem;
  }
}
```